soaks/common/configs/splunk_hec_gen.yaml

```diff
@@ -0,0 +1,14 @@
+worker_threads: 2
+prometheus_addr: "0.0.0.0:9090"
+
+targets:
+  vector:
+    target_uri: "http://vector:8282/"
+    format: "json"
+    token: "abcd1234"
+    bytes_per_second: "100 Mb"
+    parallel_connections: 10
+    maximum_prebuild_cache_size_bytes: "256 Mb"
```

Comment:
I assume these were copied from another config. I am curious if/when we would decide to modify them. I could see some parameters, like `parallel_connections`, having a direct effect on throughput.

cc/ @blt in case you have thoughts here.

Reply:
Yeah, so this specific parameter means that the cache of random values we'll chuck at vector takes up no more than 256 Mb of space in memory. Or, thought about another way, we loop through 256 Mb worth of distinct messages. It's a reasonable pool of entropy and means we'll repeat every 2 seconds and change, per the `bytes_per_second` 

We'd want to increase the cache size if we wanted more entropy or to support very large messages, the bytes per second once vector is able to absorb the load. You can also imagine a whole where we support a "go as fast as possible" setting for the bps. 

Parallel connections is an arbitrary value here, yeah.

Reply:
Thanks for the explanation!

Reply:
No specific reason for the current value. As additional context, the `parallel_connections` settings for Lading `splunk_hec_gen` also determine the number of Splunk channels used.